app/views/entities/index.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class = "app_header">
+  <a href= "<%= categories_path %>">
+    <i class="fa-solid fa-arrow-left" id = "left-arrow"></i>
+  </a>
+  <h2> Transactions</h2>
+  <a href = "/logout" id = "logout">Logout</a>
```

Comment:
- [ ] Kindly ensure you use the given font family for the entire app as this is a requirement. The given font is `Proxima Nova Bold` and not `Arial` which is currently being used